.circleci/config.yml

```diff
@@ -0,0 +1,121 @@
+version: 2.1
+commands:
+  setup:
+    description: "Setup environment"
+    steps:
+      - run: |
+          composer self-update --1
+  install_civicrm:
+    description: "Install CiviCRM"
+    parameters:
+      build_name:
+        type: string
+        default: master
+      type:
+        type: string
+        default: drupal-clean
+      version:
+        type: string
+        default: master
+      url:
+        type: string
+        default: http://localhost:8080
+    steps:
+      - run: su - buildkit -c "/buildkit/bin/civibuild create << parameters.build_name >> --civi-ver << parameters.version >> --type << parameters.type >> --url << parameters.url >>"
+  install_extension:
+    description: "Install Extension"
+    parameters:
+      build_name:
+        type: string
+        default: master
+    steps:
+      - run: |
+          EXT_DIR=/buildkit/build/<< parameters.build_name >>/web/sites/default/files/civicrm/ext/$CIRCLE_PROJECT_REPONAME
+          cp /root/project $EXT_DIR -r
+          chown buildkit:buildkit $EXT_DIR -R
+          cd $EXT_DIR
+          su - buildkit -c "cd /buildkit/build/<< parameters.build_name >>/web && cv en $CIRCLE_PROJECT_REPONAME"
+  run_civilint:
+    description: "Run civilint"
+    parameters:
+      build_name:
+        type: string
+        default: master
+    steps:
+      - run: su - buildkit -c "cd /buildkit/build/<< parameters.build_name >>/web/sites/default/files/civicrm/ext/$CIRCLE_PROJECT_REPONAME && find . -type f -not -path './vendor/*' | civilint"
+  run_phpunit:
+    description: "Run PHPUnit"
+    parameters:
+      build_name:
+        type: string
+        default: master
+    steps:
+      - run: su - buildkit -c "cd /buildkit/build/<< parameters.build_name >>/web/sites/default/files/civicrm/ext/$CIRCLE_PROJECT_REPONAME && phpunit9"
+  run_all:
+    description: "Run all steps"
+    parameters:
+      build_name:
+        type: string
+        default: master
+      type:
+        type: string
+        default: drupal-clean
+      version:
+        type: string
+        default: master
+      url:
+        type: string
+        default: http://localhost:8080
+    steps:
+      - install_civicrm:
+          build_name: << parameters.build_name >>
+          type: << parameters.type >>
+          version: << parameters.version >>
+          url: << parameters.url >>
+      - install_extension:
+          build_name: << parameters.build_name >>
+      - run_phpunit:
+          build_name: << parameters.build_name >>
+
+executors:
+  civicrm:
+    docker:
+      - image: michaelmcandrew/civicrm-buildkit
+        name: civicrm
+        environment:
+          TERM: xterm-color
+          APACHE_RUN_USER: buildkit
+      - image: mariadb:10.3
+        name: mysql
+        environment:
+          MYSQL_ROOT_PASSWORD: buildkit
+      - image: maildev/maildev
+        name: maildev
+
+jobs:
+  build:
+    executor: civicrm
+    steps:
+      - setup
+      - checkout
+      #- run_all
```

Comment:
You can remove `#` to run tests against civi@master (seems to fail at the moment).

Reply:
Yes, that's interesting, thank you. Maybe @colemanw and/or @jensschuppe could have at why that's the case.